vendor/vendor.json

```diff
@@ -21,10 +21,10 @@
 			"revisionTime": "2017-08-31T12:49:05Z"
 		},
 		{
-			"checksumSHA1": "FUhczVSvt/VQAlWfzATo1YLez2E=",
-			"path": "github.com/Financial-Times/go-fthealth/v1a",
-			"revision": "e7ccca038327a0091303a52445ec1f0fb66cb97b",
-			"revisionTime": "2017-05-25T09:50:41Z"
+			"checksumSHA1": "G8lW8pPVLieqh/MTJZkvI/i/WyM=",
+			"path": "github.com/Financial-Times/go-fthealth/v1_1",
+			"revision": "1b007e2b37b7936dfb6671fa17e5a29fd35a08ea",
+			"revisionTime": "2017-12-04T12:48:31Z"
```

Comment:
Should we pin to the `0.4.0` version (I missed this in the people-rw-neo4j review)?

Reply:
Yes I should have, however let's save time - This service is only going to live for a short time (less that two weeks hopefully) longer and we will have to create a new release and deploy. Thoughts? The orgs-rw is a little longer lived however is only used very infrequently but will die in the next 6 weeks

Reply:
Fine for this app - but for longer lived apps, we should be ensuring the vendoring is doing its job